.github/dependabot.yml

```
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

```suggestion
      interval: "daily"
      open-pull-requests-limit: 4
```

I suggest *some* limit, otherwise dependabot will open 380 PRs and we might get yelled at.

---

:+1: 
should be indented at `schedule` level though (2 spaces less)
* https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates#open-pull-requests-limit
* https://dependabot.com/docs/config-file/validator/

---

> By default, Dependabot opens a maximum of five pull requests for version updates. Once there are five open pull requests, new requests are blocked until you merge or close some of the open requests. Use open-pull-requests-limit to change this limit. This also provides a simple way to temporarily disable version updates for a package manager.

https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates#open-pull-requests-limit

The default value is 5. Is there a particular reason you would prefer 4?

---

I guess no